src/context/cooperation-context.tsx

```diff
@@ -29,15 +35,35 @@ const CooperationContext = createContext<CooperationProviderContext>(
 const CooperationProvider: FC<CooperationProviderProps> = ({ children }) => {
   const [selectedCourse, setSelectedCourse] = useState<Course | null>(null)
   const [isActivityCreated, setIsActivityCreated] = useState<boolean>(false)
+  const [isCourseAdded, setIsCourseAdded] = useState<boolean>(false)
+  const [isAddedClicked, setIsAddedClicked] = useState<boolean>(false)
+  const [currentSectionIndex, setCurrentSectionIndex] = useState(0)
```

Comment:
same with this state, seems like setCurrentSectionIndex is redundant

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/172bc738-c987-41c4-9a70-dfdde6981008)
I use this state in the CourseSectionsList component.